tests/integration-tests/tests/createami/test_createami.py

```diff
@@ -73,8 +73,8 @@ def test_invalid_config(
 
     # Test Suppression of a validator
 
-    # Get base AMI -- remarkable AMIs are not available for ARM and ubuntu2004, centos7 yet
-    if os not in ["ubuntu2004", "centos7"]:
+    # Get base AMI -- remarkable AMIs are not available for ARM and ubuntu2204, centos7 yet
+    if os not in ["ubuntu2204", "centos7"]:
```

Comment:
Why add Ubuntu22 when the AMI for remarkable is not available?

Reply:
This is a check to exclude ubuntu 22

Reply:
Sorry my Miss!